manifests/base/runtimes/deepspeed_distributed.yaml

```
@@ -0,0 +1,78 @@
+apiVersion: trainer.kubeflow.org/v1alpha1
+kind: ClusterTrainingRuntime
+metadata:
+  name: deepspeed-distributed
+  labels:
+    trainer.kubeflow.org/accelerator: gpu-tesla-v100-16gb
+spec:
+  mlPolicy:
+    numNodes: 1
+    mpi:
+      # TODO (andreyvelich): Change num proc to 1 and remove container resources after we
+      # allow to override it via TrainJob APIs.
+      numProcPerNode: 4
+      mpiImplementation: OpenMPI
+      sshAuthMountPath: /home/mpiuser/.ssh
+      runLauncherAsNode: true
+  template:
+    spec:
+      network:
+        publishNotReadyAddresses: true
+      successPolicy:
+        operator: All
+        targetReplicatedJobs:
+          - launcher
+      replicatedJobs:
+        - name: launcher
+          template:
+            metadata:
+              labels:
+                trainer.kubeflow.org/trainjob-ancestor-step: trainer
+            spec:
+              template:
+                spec:
+                  containers:
+                    - name: node
+                      image: docker.io/andreyvelichkevich/deepspeed-runtime
```

```suggestion
                      image: ghcr.io/kubeflow/trainer/deepspeed-runtime
```

Shall we change this image to github-hosted one?